### PR TITLE
Ported 'pfcmd node' command to new pfcmd framework

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Ported 'pfcmd node' command to new pfcmd framework
## Impacts

pfcmd
## NEWS file entries

New Features
++++++++++++

Enhancements
++++++++++++

Bug Fixes
+++++++++
## Delete branch after merge

NO

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Review me

**Comment by julsemaan** [comment]
node edit doesn't seem to work

**Comment by jrouzierinverse** [comment]
Please review

File: lib/pf/cmd/pf/node.pm [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+
+ pfcmd node <add|count|view|edit|delete> mac [assignments]
+
+manipulate node entries
+
+examples:
+
+  pfcmd node view all
+  pfcmd node view all order by pid limit 10,20
+  pfcmd node view pid="admin" order by pid desc limit 10,20
```
Comment (julsemaan) [inline lib/pf/cmd/pf/node.pm:16]: Please add an edit example : 
bin/pfcmd node edit 10:0b:a9:59:0c:38 status="reg"

**Comment by louismunro** [comment]
calling it without an argument returns an unhelpful error: 
    # bin/pfcmd node 
    Use of uninitialized value $action in concatenation (.) or string at /usr/local/pf/lib/pf/base/cmd/action_cmd.pm line 42.
    Usage:
         pfcmd node <add|count|view|edit|delete> mac [assignments]

**Comment by louismunro** [comment]
Seems to work otherwise.

**Comment by jrouzierinverse** [comment]
I think that is fixed in devel let me rebase and test

**Comment by jrouzierinverse** [comment]
Please review

**Comment by julsemaan** [comment]
Could you add the edit example please

**Pull request merged**